src/ui/mainwindow.cc

```diff
@@ -3626,14 +3625,14 @@ void MainWindow::scaleArticlesByCurrentZoomFactor()
 
 void MainWindow::doWordsZoomIn()
 {
-  cfg.preferences.wordsZoomLevel = cfg.preferences.wordsZoomLevel + 2;
+  ++cfg.preferences.wordsZoomLevel;
 
   applyWordsZoomLevel();
 }
 
 void MainWindow::doWordsZoomOut()
 {
-  cfg.preferences.wordsZoomLevel = cfg.preferences.wordsZoomLevel - 2;
+  --cfg.preferences.wordsZoomLevel;
 
   applyWordsZoomLevel();
 }
```

Comment:
Should we just remove this feature? This only controls the translateLine now.

There seems to be a bug when adjusting toolbar icon size that the icon size will be reset.

<img width="567" src="https://github.com/user-attachments/assets/c3cb2ad1-0e4d-4e99-acba-fca2ca70fe0e" />

Reply:
this is only used to controll the font size of translate line now.

> that the icon size will be reset.
shoud it?

Reply:
removed.